msa/black-box-tests/src/test/java/org/thingsboard/server/msa/ui/base/AbstractDriverBaseTest.java

```diff
@@ -67,7 +66,7 @@ abstract public class AbstractDriverBaseTest extends AbstractContainerTest {
 
     @SneakyThrows
```

Comment:
Please, do not use `@SneakyThrows`.
Test framework's methods likely support `throws Exception`.